### DISCUSSION TITLE
tutorials/intro-hotchocolate/14-mutation-input

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#174] [course feedback]
# tutorials/intro-hotchocolate/14-mutation-input

Design mutation inputs using common conventions.

https://www.apollographql.com/tutorials/intro-hotchocolate/14-mutation-input

**Comment by mcweglowski** [comment]
In this page code code for `Mutation.cs` where you want to return created playlist, the code: "200" does not match AddItemsToPlaylistPayload.code data type which is int, not string.

The problem occurs in code snippet after `Lastly, let's update the return value in our AddItemsToPlaylistPayload instance, specifically replacing the hard-coded playlist with the playlist variable instead.` paragraph.

**Reply by lizhennessy** [reply]
Thanks for the callout @mcweglowski ! We're shipping a fix momentarily.
